skipruntime-ts/server/package.json

```diff
@@ -20,6 +20,7 @@
   },
   "dependencies": {
     "@skipruntime/core": "0.0.13",
+    "@skipruntime/helpers": "0.0.13",
```

Comment:
Is this strictly required? Would it make sense to actually add `asLeader()`/`asFollower()` to `@skipruntime/helpers` instead?